chore: website README (#93)
hashicorp/terraform-plugin-mux: .github/CONTRIBUTING.md

## New Issue

We welcome issues of all kinds, including feature requests, bug reports, or

> documentation suggestions/contributions. Below are guidelines for well-formed issues of each

Reviewer: ```suggestion
documentation contributions. Below are guidelines for well-formed issues of each
```

   the wider Terraform ecosystem. A use case in the context of 1 provider is
   good, a use case in the context of many providers is better.

> ### Documentation Suggestions/Contributions

Reviewer: ```suggestion
### Documentation Contributions
```

   "how do servers' schemas get merged?" in mind. This helps us ensure the
   documentation is targeted, specific, and framed in a useful way.

>  - [ ] **Contribute**: This repository contains source files that power
>    versioned documentation for [terraform.io/plugin/mux](https://www.terraform.io/plugin/mux). Feel free to open a pull request with documentation changes.
>    See the [website README](../website/README.md) for additional information.

Reviewer: ```suggestion
 - [ ] **Contribute**: This repository contains the markdown files that generate versioned documentation for [terraform.io/plugin/mux](https://www.terraform.io/plugin/mux). Please open a pull request with documentation changes. Refer to the [website README](../website/README.md) for more information.
```
